dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -107,7 +109,6 @@ public final PreparedStatement createStatementWithBoundFixtureSymbols(
         String command = Options.isBindSymbols() ? parseCommandText(commandText) : commandText;
         PreparedStatement cs = getConnection().prepareStatement(
                 command);
-
```

Comment:
> (BTW, do we have any preferred style for when we want blank lines?)

When it helps for readability... e.g. it's a new thought (similar to new paragraph in a natural language text).